src/index.test.ts

```diff
@@ -157,6 +157,30 @@ describe('post()', () => {
     // assert error is empty
     expect(error).toBe(undefined);
   });
+
+  it('supports optional requestBody', async () => {
+    const mockData = { status: 'success' };
+    const client = createClient<paths>();
+    fetchMocker.mockResponse(() => ({ status: 201, body: JSON.stringify(mockData) }));
+
+    // assert omitting `body` doesn’t raise a TS error (testing the response isn’t necessary)
+    await client.post('/create-tag/{name}', {
+      params: { path: { name: 'New Tag' } },
+    });
+
+    // assert providing `body` with correct schema doesn’t raise a TS error
+    await client.post('/create-tag/{name}', {
+      params: { path: { name: 'New Tag' } },
+      body: { description: 'This is a new tag' },
```

Comment:
This is the test in question. Note that although there is no test assertion here, this **WILL** throw an error from `npm run lint` if it’s invalid